singularity-eos/eos/eos_spiner.hpp

```diff
@@ -1216,8 +1212,10 @@ Real SpinerEOSDependsRhoT::lRhoFromPlT_(const Real P, const Real lT,
     lambda[Lambda::lRho] = lRho;
     lambda[Lambda::lT] = lT;
   }
-  status_ = status;
-  whereAmI_ = whereAmI;
+  if (memoryStatus_ != DataStatus::OnDevice) {
+    status_ = status;
+    whereAmI_ = whereAmI;
+  }
```

Comment:
Thanks for catching this. My great shame it was left in the code after I thought I found all these.